make/apis.mk

```diff
@@ -0,0 +1,31 @@
+# Copyright 2023 D2iQ, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+export CAPA_REPO := https://github.com/kubernetes-sigs/cluster-api-provider-aws.git
+export CAPA_VERSION := v2.2.2
+export CAPA_API_PACKAGE_NAME := sigs.k8s.io/cluster-api-provider-aws/v2/api/
```

Comment:
should we limit syncing to api versions that we use? for ex. `v1beta2` here?

Reply:
Good idea, done https://github.com/d2iq-labs/capi-runtime-extensions/pull/190/commits/957e6666082c442da169f49870605c5063ba06da